docs/cudf/source/guide-to-udfs.ipynb

```diff
@@ -22,7 +22,7 @@
     "- CuPy NDArrays\n",
```

Comment:
<p>We publish docs separately for different cuDF versions, so it's not strictly necessary to include versioning information in the docs. Ditto below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/8810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>